src/test/java/DataDriven/framework/FormModelObject.java

```diff
@@ -0,0 +1,68 @@
+package DataDriven.framework;
+
+public class FormModelObject{
```

Comment:
This models student data but the name "FormModel" implies that this can only be used with a form. It should not know how it will be used. I would rename to "Student" or "StudentData". We can omit "Object" from the name because almost all classes will become and object.